server/stt/stt_events_ml.py

```diff
@@ -217,7 +217,18 @@ def set_location_details(self, item, location_xml, notes):
         try:
             name = location_xml.find(self.qname("name")).text
             location["name"] = name
-            location["unique_name"] = name
+            address_xml = location_xml.find(self.qname("POIDetails")).find(
```

Comment:
this is also used later and seems like it might be empty, then this whole part would fail and there would be no `unique_name`